src/components/SelectionList/BaseSelectionList.js

```diff
@@ -259,33 +280,32 @@ function BaseSelectionList({
 
     const renderItem = ({item, index, section}) => {
         const normalizedIndex = index + lodashGet(section, 'indexOffset', 0);
-        const isDisabled = section.isDisabled;
+        const isDisabled = section.isDisabled || item.isDisabled;
         const isItemFocused = !isDisabled && focusedIndex === normalizedIndex;
         // We only create tooltips for the first 10 users or so since some reports have hundreds of users, causing performance to degrade.
         const showTooltip = normalizedIndex < 10;
 
-        if (canSelectMultiple) {
-            return (
-                <UserListItem
-                    item={item}
-                    isFocused={isItemFocused}
-                    onSelectRow={() => selectRow(item, true)}
-                    onDismissError={onDismissError}
-                    showTooltip={showTooltip}
-                />
-            );
-        }
-
         return (
-            <RadioListItem
+            <BaseListItem
                 item={item}
                 isFocused={isItemFocused}
                 isDisabled={isDisabled}
+                showTooltip={showTooltip}
+                canSelectMultiple={canSelectMultiple}
                 onSelectRow={() => selectRow(item, true)}
+                onDismissError={onDismissError}
             />
         );
     };
 
+    const onLayout = () => {
```

Comment:
We should avoid naming event handlers like `onSomething` - it's even in PR Author's Checklist:

> I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. toggleReport and not onIconClick)

So the name should define what the handler does.

Reply:
True, force of habit :)

Changing to `scrollToFocusedIndex`